db/schema.rb

```diff
@@ -75,6 +75,7 @@
     t.boolean "memorial", default: false, null: false
     t.bigint "moved_to_account_id"
     t.string "featured_collection_url"
+    t.jsonb "fields"
```

Comment:
If `fields` is JSONB shouldn't be a Hash and not an Array? 

See report on https://discourse.joinmastodon.org/t/error-on-profile-edit-for-single-user-nomethoderror-undefined-method-for-hash/1563